### PR TITLE
Fix missing outputs in Gamma tute JPN

### LINKED ISSUE #1662: Outputs from new tutorial not being rendered
Normally the docs server should run the notebook and render the outputs, that hasn't happened for the new notebook though:

https://docs.pymedphys.com/lib/tutes/gamma/gamma_tutorial.html

### PR DESCRIPTION
**Pull request by Matthew-Jennings**
Closes #1662

### CODE REVIEW

**Comment by Matthew-Jennings** [comment]
I believe this is now fixed.

---------------------

The original "passing" Windows docs test actually complained of a bad kernelspec [here](https://github.com/pymedphys/pymedphys/runs/6835612399?check_suite_focus=true#step:44:88). This was not present before.

I've changed the relevant lines in the gamma tute notebook metadata [here](https://github.com/pymedphys/pymedphys/blob/b573874a05d8929b8f93a2370521835ec5775499/lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb?short_path=8cdecde#L556-L560) to match [those in a working notebook](https://github.com/pymedphys/pymedphys/blob/b573874a05d8929b8f93a2370521835ec5775499/lib/pymedphys/docs/lib/howto/gamma/effect-of-noise.ipynb?short_path=76f3583#L366-L370)

This will likely fix the issue since the CLI is now [no longer complaining about the kernel](https://github.com/pymedphys/pymedphys/runs/6848434922?check_suite_focus=true#step:44:92). I can't tell for sure yet because the tute was showing the outputs fine when I built the docs locally, and there was no similar error noted in the build console output.

_Shall we try merging?_

**Review by SimonBiggs — APPROVED** [review]

**Comment by SimonBiggs** [comment]
> Shall we try merging?

Yup, go for it :)

**Comment by SimonBiggs** [comment]
> The original "passing" Windows docs test actually complained of a bad kernelspec

Once we have sorted out all of the warnings with building the docs we can set it to raise an error on warnings being raised. That way, issues like this shouldn't go unnoticed.

**Comment by Matthew-Jennings** [comment]
> Yup, go for it :)

Sorry, any idea what's going on with the `propagate` stuff before I dive into that particular rabbi hole?

**Comment by SimonBiggs** [comment]
Yup @nlambriICH added a line to support forks. I suspect that line just needs a brackets around it. Currently it says:

```
git fetch origin $BRANCH || git remote add upstream_fork "https://github.com/${REPO}.git" && git fetch upstream_fork
```

I suspect it may need to say:

```
git fetch origin $BRANCH || (git remote add upstream_fork "https://github.com/${REPO}.git" && git fetch upstream_fork)
```

**Comment by SimonBiggs** [comment]
Or instead, have less logic entirely and have something like:

```
# untested
git remote set-url origin "https://github.com/${REPO}.git"
git fetch origin $BRANCH
```

File: .github/workflows/library.yml [hunk 1]
```diff
@@ -364,8 +364,8 @@ jobs:
           git config --local user.email "propagate-bot@pymedphys.com"
           git config --local user.name "PyMedPhys Propagate Bot"
 
-          git fetch origin $BRANCH || git remote add upstream_fork "https://github.com/${REPO}.git" && git fetch upstream_fork
-          git checkout $BRANCH
```
Comment (SimonBiggs) [inline .github/workflows/library.yml:368]: Checkout will still be needed I suspect

File: .github/workflows/library.yml [hunk 2]
```diff
@@ -364,8 +364,8 @@ jobs:
...
+          git remote set-url origin "https://github.com/${REPO}.git"
+          git fetch origin $BRANCH
```
Comment (SimonBiggs) [inline .github/workflows/library.yml:369]:
```suggestion
git checkout $BRANCH
```

File: .github/workflows/library.yml [hunk 3]
```diff
@@ -366,6 +366,7 @@ jobs:
 
           git remote set-url origin "https://github.com/${REPO}.git"
           git fetch origin $BRANCH
+git checkout $BRANCH
```
Comment (SimonBiggs) [inline .github/workflows/library.yml:369]: Haha... Oops... Fixing things on the phone... The lack of indent here will break most of the CI :/

**Comment by SimonBiggs** [comment]
By the way, the online CI is able to be rerun through the GitHub interface (it can also be cancelled as well to free up CI resources).

**Comment by Matthew-Jennings** [comment]
> By the way, the online CI is able to be rerun through the GitHub interface (it can also be cancelled as well to free up CI resources).

Thanks! I'd forgotten that.

**Comment by Matthew-Jennings** [comment]
So I accidentally cancelled the wrong set of runs and it will; only allow me to re-run one at a time! :(

Dummy will be much faster - this has been taking forever

**Comment by Matthew-Jennings** [comment]
Remind me to never run `propagate` locally again!

**Comment by SimonBiggs** [comment]
> Remind me to never run `propagate` locally again!

Why's that? :)

**Comment by Matthew-Jennings** [comment]
> > Remind me to never run `propagate` locally again!
> 
> Why's that? :)

It was changes introduced by running `propagate` on my Windows PC that caused the windows doctests to start failing again (with a different error than the original).

**Comment by SimonBiggs** [comment]
That's weird :/... Not sure how it could do that...

**Pull request merged**